global.d.ts

```diff
@@ -0,0 +1 @@
+declare function svgJapan(options: any): any;
```

Comment:
- Looks good, but if possible please try to include a different type to any

Reply:
Hi Kamil. Thank you for your advice. I fixed the global.d.ts file and specified the type from any.